### ISSUE TITLE
Add NOT NULL constraint to descriptions

### ISSUE THREAD

**Issue by lnschroeder** [KarnerTh/mermerd#42]
**Motivation**
It would be great if it was possible to to visualize `NOT NULL` constraints.

**Solution**
Unfortunately, Mermaid does not support special characters (such as asterisks) in the name column. 
Instead the comment column could be used (as suggested [here](https://github.com/orgs/mermaid-js/discussions/3718#discussioncomment-6222401)). 

To toggle this, `notNull` could be added to the `showDescription` flag e.g.
```yml
showDescriptions:
  - columnComments
  - enumValues
  - notNull
```

![image](https://github.com/KarnerTh/mermerd/assets/33390735/0564001a-135a-4b9f-92cb-4b002d8a1e95)

I am happy to make a PR for this. 

All feedback is welcome :)

**Comment by KarnerTh** [comment]
Totally agree!

Feel free to create a PR :) I am on vacation, but as soon as I am back I will review it

**Comment by KarnerTh** [comment]
Change is available in version [0.9.0](https://github.com/KarnerTh/mermerd/releases/tag/v0.9.0).

Thanks for your contribution 🙌🏼
